src/api/types/cron/index.ts

```diff
@@ -0,0 +1,13 @@
+interface ApplicationWhereUpdate {
+  id: string;
+}
+
+interface ApplicationDataUpdate {
+  status: string;
+  previousStatus: string;
+}
+
+export interface InactiveApplicationUpdate {
+  where: ApplicationWhereUpdate;
+  data: ApplicationDataUpdate;
+}
```

Comment:
Should we maybe prefix these types with `Cron` e.g 

```js
CronApplicationWhereUpdate
CronApplicationDataUpdate
CronApplicationInactiveupdate
```

Just thinking so it's clear that it's specifically for a CRON job